src/rev_ai/apiclient.py

```diff
@@ -260,13 +280,26 @@ def get_captions(self, id_, filename=None, filepath=None):
             url, headers={'Accept': self.rev_captions_content_type})
         response.raise_for_status()
 
-        if filename:
-            path = os.path.join(filepath or '', filename+'.txt')
-            with open(path, 'w+') as f:
-                f.write(response.text)
-
         return response.text
 
+    def get_captions_as_stream(self, id_):
+        """Get the captions output of a specific job and return it as a plain text stream
```

Comment:
Do we have an issue to add vtt output format to python SDK?

Reply:
I will file one. It will likely be done along with AISDK-52 which will abstract away the requests package